core/templates/dev/head/pages/preferences-page/preferences-page.directive.html

```diff
@@ -214,6 +214,20 @@
       </div>
     </div>
 
+    <hr ng-if="$ctrl.canRemoveAccount" >
```

Comment:
Nit: drop the space before `>` at the end

Reply:
Done.